src/services/latex.js

```diff
@@ -29,9 +29,9 @@ var latexMathParser = (function() {
   var symbol = regex(/^[^${}\\_^]/).map(function(c) { return VanillaSymbol(c); });
 
   var controlSequence =
-    regex(/^[^\\a-eg-zA-Z]/) // hotfix #164; match MathBlock::write
+    regex(/^[^\\a-eg-zA-Z|]/) // hotfix #164; match MathBlock::write
```

Comment:
yes, as a control sequence. However, the standalone | character will be parsed y the VanillaSymbol parser above.